crates/runtime/src/env/default/memory.rs

```diff
@@ -24,4 +24,6 @@ impl EnvTypes for DefaultMemEnvTypes {
     type AppAddressCompute = default::DefaultAppAddressCompute;
 
     type TemplateHasher = default::DefaultTemplateHasher;
+
+    type Pricer = svm_gas::resolvers::StandardPriceResolver;
```

Comment:
For now that's OK. But It seems more likely that will have more `PriceResolver` in the future 
(see the above comment about versioning).

I think it'd be better to inject it as a dependency to the `Runtime`

Reply:
This will be done while working on the issue that we'll create now, correct? Not in this PR?